diff -Naur xdg-utils-1.1.1/scripts/xdg-open.in xdg-utils-1.1.1-lca_support/scripts/xdg-open.in
--- xdg-utils-1.1.1/scripts/xdg-open.in	2015-10-05 21:59:19.000000000 +0300
+++ xdg-utils-1.1.1-lca_support/scripts/xdg-open.in	2016-04-20 12:27:30.394886947 +0300
@@ -348,6 +348,13 @@
 
         check_input_file "$file"
 
+        if which lca-tool 2>/dev/null 1>&2; then
+            lca-tool --file --triggerdefault "$file"
+            if [ $? -eq 0 ]; then
+                exit_success
+            fi
+        fi
+
         if [ -n "$DISPLAY" ]; then
             filetype=`xdg-mime query filetype "$file" | sed "s/;.*//"`
             open_generic_xdg_mime "$file" "$filetype"
@@ -368,6 +375,13 @@
         fi
     fi
 
+    if which lca-tool 2>/dev/null 1>&2; then
+        lca-tool --scheme --triggerdefault "$1"
+        if [ $? -eq 0 ]; then
+            exit_success
+        fi
+    fi
+
     if [ -n "$BROWSER" ]; then
         open_envvar "$1"
     fi
