Description: add lca-tool support
Author: Fathi Boudra <fathi.boudra@nokia.com>
--- a/scripts/xdg-open
+++ b/scripts/xdg-open
@@ -434,6 +434,14 @@ open_generic()
             fi
         fi
 
+        if [ -f /etc/meego-release ] &&
+            which lca-tool 2>/dev/null 1>&2; then
+            lca-tool --file --triggerdefault "$file"
+            if [ $? -eq 0 ]; then
+                exit_success
+            fi
+        fi
+
         if mimeopen -v 2>/dev/null 1>&2; then
             mimeopen -L -n "$file"
             if [ $? -eq 0 ]; then
