From 131cfba7421ca12f2d98063332a29507dd47409d Mon Sep 17 00:00:00 2001
From: Vesa-Matti Hartikainen <vesa-matti.hartikainen@jollamobile.com>
Date: Tue, 16 Apr 2013 17:49:44 +0300
Subject: [PATCH] open protocols using lca-tool when possible

---
 scripts/xdg-open |    7 +++++++
 1 file changed, 7 insertions(+)

Index: xdg-utils/scripts/xdg-open
===================================================================
--- xdg-utils.orig/scripts/xdg-open
+++ xdg-utils/scripts/xdg-open
@@ -716,6 +716,13 @@ open_generic()
         fi
     fi
 
+    if which lca-tool 2>/dev/null 1>&2; then
+        lca-tool --scheme --triggerdefault "$1"
+        if [ $? -eq 0 ]; then
+            exit_success
+        fi
+    fi
+
     open_generic_xdg_x_scheme_handler "$1"
 
     IFS=":"
