From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Raine Makelainen <raine.makelainen@jolla.com>
Date: Tue, 23 Jan 2018 20:10:25 +0200
Subject: [PATCH] lca support

---
 scripts/xdg-open.in | 14 ++++++++++++++
 1 file changed, 14 insertions(+)

diff --git a/scripts/xdg-open.in b/scripts/xdg-open.in
index feacc1f..4943163 100755
--- a/scripts/xdg-open.in
+++ b/scripts/xdg-open.in
@@ -482,6 +482,13 @@ open_generic()
 
         check_input_file "$file"
 
+        if which lca-tool 2>/dev/null 1>&2; then
+            lca-tool --file --triggerdefault "$file"
+            if [ $? -eq 0 ]; then
+                exit_success
+            fi
+        fi
+
         if has_display; then
             filetype="$(xdg-mime query filetype "$file" | sed "s/;.*//")"
             # passing a path a url is okay too,
@@ -504,6 +511,13 @@ open_generic()
         fi
     fi
 
+    if which lca-tool 2>/dev/null 1>&2; then
+        lca-tool --scheme --triggerdefault "$1"
+        if [ $? -eq 0 ]; then
+            exit_success
+        fi
+    fi
+
     if has_display; then
         open_generic_xdg_x_scheme_handler "$1"
     fi
